docs/best_practice/s3_setup.txt

```diff
@@ -0,0 +1,135 @@
+.. highlight:: yaml
+.. _s3_setup:
+
+=========================
+Crate Setup for Amazon S3
+=========================
+
+You can create CRATE *Snapshots* on the `Amazon S3`_ (Amazon Simple Storage
+Service). For this, you need to register the `AWS` plugin with CRATE.
+
+
+Basic Configuration
+===================
+
+In order to support *Snapshot* and *Restore* to the `Amazon S3`_ service, the
+`AWS` plugin needs to be enabled. This is done, by enabling the **cloud setting
+in the ``crate.yml`` file::
+
+  cloud.enabled: true
```

Comment:
maybe we should not call this option 

`cloud.enabled`

but something else? like

`plugin.aws.enabled`?